Makefile

```diff
@@ -0,0 +1,62 @@
+NAME        := cart
+VERSION     := 0.0.1
```

Comment:
Does CaRT currently use a version for shared libraries?  It would be good to tie these together at some point.

Reply:
They are currently .0.0.1.  That's why I set the version here to 0.0.1.